addons/hr/views/res_users.xml

```diff
@@ -137,70 +67,17 @@
                                     <field name="private_zip" placeholder="ZIP" class="o_address_zip" readonly="not can_edit"/>
                                     <field name="private_country_id" placeholder="Country" class="o_address_country" options='{"no_open": True, "no_create": True}' readonly="not can_edit"/>
                                 </div>
-                                <field name="private_email" string="Email" readonly="not can_edit"/>
```

Comment:
A lot of fields from here :p please ensure we removed them if unused and only displayed here